lblod/ember-rdfa-editor-lblod-plugins
# recreate mappings uri

### Overview
When recreating the variable uris we weren't recreating the mapping, which can result in unwanted behaviour

##### connected issues and PRs:
Maybe GN-5232


### Setup
None

### How to test/reproduce
Copy and paste a variable notice that the mapping uri also gets recreated, both if the uri is a template uri or a real uri

### Challenges/uncertainties
I'm looking at this pluggin and it's weird because I thought it was more general, than just recreating variable uris



### Checks PR readiness
